src/command/FileCommands.cxx

```diff
@@ -233,3 +239,103 @@ handle_read_comments(Client &client, Request args, Response &r)
 
 	gcc_unreachable();
 }
+
+InputStreamPtr
+find_stream_art(const char *directory, Mutex &mutex, Cond &cond)
+{
+    static constexpr char const * art_names[] = {
```

Comment:
Wrong indent - everything should be indented with tabs, not 4 spaces. See https://www.musicpd.org/doc/developer/code_style.html